### ISSUE TITLE
Update to PHP7 and ext-mongodb

### ISSUE THREAD

**Issue by rubenrua** [campusdomar/PuMuKIT2#484]

**Comment by rubenrua** [comment]
Waiting for doctrine/mongodb-odm/pull/1553

**Comment by q0re** [comment]
I tried a lot, but i could not get it to work with php7.

What i have done so far:

1. Update composer file and delete:

`"doctrine/mongodb-odm": "1.0.x-dev#328c1eba840a4f3b81d3a28211b3f1ce5445ceaf",`
`"doctrine/mongodb-odm-bundle": "3.0.x-dev#1ac417f1e1374474778707f9f0d7d82b19d17922"`

2. delete the post-install-cmd script as well

3. config alcaeus mongo-php-adapter:

4. follow the installation guide (set permissions, ...)

`composer config "platform.ext-mongo" "1.6.16" && composer require alcaeus/mongo-php-adapter`

**Result**
app_dev.php still throws the Error `Compile Error: Cannot use 'Int' as class name as it is reserved`

I used grep to make sure that the complete vendor folder is not using the `@Int` Annotation. Any ideas? Maybe a autoload is loading some references?
